ha/util/conf_store.py

```diff
@@ -110,3 +121,93 @@ def set_cluster_cardinality(self, index):
         cluster_cardinality_key = CLUSTER_CARDINALITY_KEY
         cluster_cardinality_value = {CLUSTER_CARDINALITY_NUM_NODES: num_pods, CLUSTER_CARDINALITY_LIST_NODES : watch_pods }
         self._confstore.update(cluster_cardinality_key, json.dumps(cluster_cardinality_value))
+
+    def get_cvg_list(self, index, node_id):
+        """
+        Return list of CVG's for any given node.
+        Args:
+            index(str): index of conf file
+            node_id (str): node id
+        Returns:
+            list: list of CVG's
+
+        >>> get_cvg_list(index, '21d6291109304485b3daff43a06cff77')
+        ['cvg-01', 'cvg-02']
+        """
+        try:
+            cvg_list = []
+            cvg_count = Conf.get(index, GconfKeys.CVG_COUNT.value.replace("(node_id)", node_id))
+            cvg_list = [Conf.get(index, GconfKeys.CVG_NAME.value.replace("(node_id)", node_id)
+                            .replace("(cvg_index)", str(c))) for c in range(cvg_count)]
+            return cvg_list
+        except Exception as e:
+            Log.error(f"Unable to fetch CVG list from GConf. Error {e}")
+            raise Exception(f"Unable to fetch CVG list. Error {e}")
```

Comment:
Conf will return `cvg_count` as None for non-existing node_id. Then we will see `TypeError: None type object` from line 141. At last, the error message won't be relevant to node_id.
If cvg_count is None from Conf, raise Exception with appropriate message.

Reply:
Raised Exception with appropriate message.